### PR TITLE
Fix target framework for System.Text.Encodings.Web.Tests

### PR DESCRIPTION
**Pull request by Maxwe11**

### CODE REVIEW

**Comment by Maxwe11** [comment]
@dotnet-bot test this please

**Comment by mellinoe** [comment]
Did we just forget to update this when we moved to project.json? Looks correct now to me.

**Comment by Maxwe11** [comment]
@mellinoe this project also didn't have `packages.config` thus moving to `project.json` passed beside.
[Some other projects](https://github.com/dotnet/corefx/search?utf8=%E2%9C%93&q=TargetFrameworkVersion) also targeting full .NET instad of .NET Core but I think thats by design?

File: src/System.Text.Encodings.Web/tests/project.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "dependencies": {
+    "System.Reflection": "4.0.10-beta-22809",
+    "System.Reflection.Extensions": "4.0.0-beta-22809",
+    "System.Runtime": "4.0.20-beta-22809",
+    "System.Linq": "4.0.0-beta-22809",
+    "System.Text.Encoding.Extensions": "4.0.10-beta-22809"
+  }
+}
```
Comment (stephentoub) [inline src/System.Text.Encodings.Web/tests/project.json]: Nit: these files are missing newlines at the end.

**Comment by stephentoub** [comment]
LGTM. @KrzysztofCwalina ?

File: src/System.Text.Encodings.Web/tests/project.json (hunk 1 above)
Comment (Maxwe11) [inline src/System.Text.Encodings.Web/tests/project.json]: fixed

**Comment by KrzysztofCwalina** [comment]
Looks good to me.

**Comment by KrzysztofCwalina** [comment]
@Maxwe11, thanks for doing this fix!

**Pull request merged**